src/main/java/edu/tamu/scholars/middleware/discovery/controller/IndividualSearchController.java

```diff
@@ -56,9 +56,9 @@ public ResponseEntity<PagedModel<IndividualResource>> search(
         HighlightArg highlight,
         @PageableDefault(page = 0, size = 10, sort = "id", direction = ASC) Pageable page
     ) {
-    // @formatter:on
         return ResponseEntity.ok(discoveryPagedResourcesAssembler.toModel(repo.search(query, facets, filters, boosts, highlight, page), assembler));
     }
+    // @formatter:on
```

Comment:
I've never been a fan of editor specific helpers in code.

Reply:
Prevents auto format changing format.